hieradata/production.yaml

```diff
@@ -102,6 +102,9 @@ govuk_jenkins::job::network_config_deploy::environments:
   - 'carrenza-production-dr'
   - 'skyscape-production'
 
+govuk_jenkins::job::content_performance_manager:
```

Comment:
This should be `govuk_jenkins::job::content_performance_manager::rake_import_all_content_items_frequency: '0 2 * * *'` (a single line, otherwise it thinks it's a hash when it's really a parameter string)